odonto/models.py

```diff
@@ -187,3 +187,119 @@ class Fp17Declaration(models.EpisodeSubrecord):
 
     signature                  = fields.CharField(max_length=255, blank=True, null=True)
     signature_date             = fields.DateField(blank=True, null=True)
+
+
+"""
+WIP: "Data flow" models. (Not to be merged)
+"""
+
+class Provider(models.Model):
+    name = models.CharField(max_length=255)
+
+    contract_number = models.IntegerField(unique=True)
+    provider_number = models.IntegerField(unique=True)
+
+    created = models.DateTimeField(default=timezone.now)
+
+    class Meta:
+        ordering = ('name',)
+        get_latest_by = 'created'
+
+    def __str__(self):
+        return "pk={0.pk} name={0.name!r}".format(self)
+
+
+class Performer(models.Model):
+    provider = models.ForeignKey(
+        'Profiver',
+        on_delete=models.CASCADE,
+        related_name='performers',
+    )
+
+    # ??
+    user = models.ForeignKey(
+        'auth.User',
+        on_delete=models.CASCADE,
+        related_name='',
```

Comment:
what happens when you put in an empty related_name?

Reply:
It fails. (These are super draft models!)